frontend/src/app/main/ui/workspace/tokens/management/tooltip.cljs

```diff
@@ -0,0 +1,148 @@
+(ns app.main.ui.workspace.tokens.management.tooltip
```

Comment:
Perhaps this module could be `app.main.data.workspace.tokens.format`, since it's not directly part of the UI components, and it's a generic functionality that may be used from other than tooltips.